spec/controllers/api/v1/companies_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CompaniesController, type: :controller do
+
+  describe "GET #show" do
+    let(:response_body) { JSON.parse(response.body) }
+    let(:company) { create(:company) }
+
```

Comment:
I haven't seen the let syntax to set up variables in Rails before. Looks very much like JavaScript. Is there a specific reason you used this instead of the "Ruby" way?
Does it accomplish something different?